front-end/peer-prep/src/components/Auth/VerificationPage/styles.tsx

```diff
@@ -0,0 +1,91 @@
+export const VerificationPageContainerStyle = {
```

Comment:
Should start with small letter, verificationPageContainerStyle. Similarly for line 23.

Reply:
thanks for pointing out, made the necessary changes